addons/point_of_sale/static/src/js/models.js

```diff
@@ -425,11 +425,22 @@ exports.PosModel = Backbone.Model.extend({
         loaded: function(self, categories){
             self.db.add_categories(categories);
         },
+    },{
+        model: 'product.packaging',
```

Comment:
you probably should get the packaging after the products to only take the packagin linked to products available in POS.